meetuppizza/settings/base.py

```diff
@@ -29,6 +31,15 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = False
 
+MEETUP_KEY = os.getenv("MEETUP_KEY")
```

Comment:
Beware of using `os.getenv` directly here. What happens if one of these environment variables isn't defined? Where will the error message be thrown, and how useful will it be? If you take a look at Two Scoops, there's a widely used idiom designed to help solve this problem.